### PR TITLE
Unprivate Service.prototype.getToken_

### PR DESCRIPTION
**Pull request by oshliaer**
I moved to public `Service.prototype.getToken_`.
I also commited package-lock.json file and build a new dist https://stackoverflow.com/questions/44206782

Should I split this into two pull requests?

### CODE REVIEW

File: dist/OAuth2.gs [hunk 1]
```diff
@@ -467,7 +467,7 @@ Service_.prototype.getRedirectUri = function() {
 Service_.prototype.getTokenFromResponse_ = function(response) {
   var token = this.parseToken_(response.getContentText());
   var resCode = response.getResponseCode();
-  if ( !(resCode >= 200 && resCode < 300) || token.error) {
+  if ( resCode < 200 || resCode >= 300 || token.error) {
```
Comment (oshliaer) [inline dist/OAuth2.gs]: This happened automatically.

**Comment by erickoledadevrel** [comment]
I would prefer if this PR just included the changes to Service.gs, with the dist changes later.

**Comment by oshliaer** [comment]
@erickoledadevrel , it seems that the reset to the previous commit is appropriate.

**Comment by erickoledadevrel** [comment]
Thanks so much!

**Comment by oshliaer** [comment]
Your patience helps me. Thank you too.

**Pull request merged**